src/control/server/ctl_storage.go

```diff
@@ -216,10 +216,12 @@ func (c *StorageControlService) ScanNvme() (types.NvmeControllers, error) {
 // ScanScm scans locally attached modules and returns list directly.
 //
 // Suitable for commands invoked directly on server, not over gRPC.
-func (c *StorageControlService) ScanScm() (types.ScmModules, types.PmemDevices, error) {
+func (c *StorageControlService) ScanScm() ([]scm.Module, []scm.Namespace, error) {
```

Comment:
Doesn't this mean that the storage scan cmd needs to handle stringifying these results now?

Reply:
yes, Will add types and stringer interface